common/lib/dependabot/dependency.rb

```diff
@@ -61,6 +62,10 @@ def top_level?
       requirements.any?
     end
 
+    def removed?
+      @removed
```

Comment:
💡 What do you think about this explicitly returning a boolean? I'd expect that based on the `?` ending of the method name.

````suggestion
    !@removed.nil?
````

Reply:
This should already return a boolean since I've defaulted it to false at https://github.com/dependabot/dependabot-core/pull/5549/files#diff-75ed2e4bde042ddd4c61aeb79efee8d861ecae1351d692fb94607b581d647f05R44

I just wanted the `?` on the accessor. Would you do that another way?

Reply:
I'd reach for the `?` method as well 😄 I'd personally make sure to return `true`/`false` because it's a [pretty established convention](https://github.com/rubocop/ruby-style-guide#predicate-methods-suffix). The other option is `!!@removed`, but I think Rubocop favors this approach.